using_images/other_images/jenkins.adoc

```diff
@@ -64,6 +79,71 @@ $ oc new-app -e \
     openshift/jenkins-1-centos7
 ----
 
+If `*JENKINS_PASSWORD*` is not specified on the `oc new-app` invocation, the password is an auto-generated, random string.  It will be reported in the new-app
+output.  If you lose this value you can retrieve it by looking at the environment variable values defined on the Jenkins deployment configuration in the web console or by
+    running:
+
+        $ oc env dc/jenkins --list | grep JENKINS_PASSWORD
+
+ifdef::openshift-origin[]
+==== {product-title} OAuth authentication
+
+{product-title} OAuth authentication will be used when it is explicitly configured in the `*Configure Global Security*` panel, or it will be used automatically for the {product-title} Jenkins images
+when OpenShift Login plugin determines that it is running in a pod and the `*OPENSHIFT_ENABLE_OAUTH*` environment variable on the Jenkins `*Deployment Config*` is set to anything other than `false`.  The plugin will retrieve the configuration information needed
+either from data seeded into the pod, or by interacting with the {product-title} API server via REST invocations.  There are configuration fields available for the plugin, but they should only be required for perhaps
+development testing or atypical configurations when you want to access a different {product-title} installation for OAuth than the installation hosting Jenkins.
+
+The login process is comprised of a series of HTTP redirects.  Rather than the traditional Jenkins login page, you will first be directed to an {product-title} login page, similar to what you see when you log into the web console.  If login
+is successful, you will be redirected to the page within the Jenkins UI you are trying to access.
+
+Valid user / password combinations are controlled by the identity provider in use with {product-title}.  So, for example,  if the default identity provider `Allow All` is used, you can provide any
+non-empty string as the password for `admin` or any other user name of your choosing.  If one of the other identity providers is running, valid user/password combinations stored with your identity provider must be used to login.
```

Comment:
OK, I was able to confirm that the user ID that created the project does in fact have the admin role in the OOTB scenario.

I suppose it is safe to assume that if say a user with OpenShift Online instantiates the jenkins template in his project, the same will apply.

I'll rework the document to encourage the use of the user that he logged into openshift with / created the project under.